TokenAdministrationApi/Startup.cs

```diff
@@ -114,7 +114,7 @@ private static void ConfigureDbContext(IServiceCollection services)
             var connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING");
 
             services.AddDbContext<TokenDatabaseContext>(
-                opt => opt.UseNpgsql(connectionString));
+                opt => opt.UseNpgsql(connectionString ?? "connstr"));
```

Comment:
Is `connstr` the value we want to use when no connection string was found e.g maybe in the tests 🤔 or is this to stop `Null` from being passed in as an argument?

Reply:
When you run migrations it needs to build the project first but the build fails because this is null if you don't first set an env variable so it's to stop the build failing without a connection string.